docs/build/backends.md

```diff
@@ -0,0 +1,33 @@
+!!! note
+    This documentation page needs some more details and explanations.
+    It is currently a work in progress.
+
+To decouple the building of a conda package from pixi we provide something what are called build backends.
+These are essentially executables following a specific protocol that is implemented for both pixi and the build backend.
+This also allows for decoupling of the build backend from pixi and it's manifest specification.
+
+The backends we are currently developing are available in the following [conda channel](https://prefix.dev/channels/pixi-build-backends).
+And are being developed in the [pixi-build-backends](https://github.com/prefix-dev/pixi-build-backends) repository.
+
+### Installation
+When looking at the following build-section:
+
+```toml
+[build-system]
+build-backend = { name = "pixi-build-cmake", version = "*" }
+channels = [
+  "https://prefix.dev/pixi-build-backends"
```

Comment:
```suggestion
  "https://prefix.dev/pixi-build-backends",
```

Or even better, get it from the tested files.

Reply:
Indeed, thanks for the suggestion!